src/bundle/DependencyInjection/Configuration/Parser/LocationIds.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUiBundle\DependencyInjection\Configuration\Parser;
+
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\AbstractParser;
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\SiteAccessAware\ContextualizerInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+class LocationIds extends AbstractParser
+{
+    /**
+     * Adds semantic configuration definition.
+     *
+     * @param NodeBuilder $nodeBuilder Node just under ezpublish.system.<siteaccess>
+     */
+    public function addSemanticConfig(NodeBuilder $nodeBuilder)
+    {
+        $nodeBuilder
+            ->arrayNode('location_ids')
+                ->info('System locations id configuration')
+                ->children()
+                    ->scalarNode('content')->isRequired()->end()
+                    ->scalarNode('media')->isRequired()->end()
+                    ->scalarNode('users')->isRequired()->end()
```

Comment:
As far as I remember we dynamically create corresponding admin siteaccess for every defined siteacces in the system. The best approach would be to define `location_ids` per groups created by siteaccess + its admin SA. We need to describe it in the documentation  though. 

cc @DominikaK

Reply:
@alongosz not documented yet, but it is on my to-do list. I will need information on the use cases where a user may want to do anything to the config (and should they?)

Reply:
There is a default configuration coming with the AdminUI which will work with our meta and demos. Configuration change is only needed when location IDs for Home, Media and Users differs from meta.

Reply:
The most common use-case for this would be multi-site on one repository, where each site would have its own content, media and user root. Without going into details, allowing to change this probably requires more changes than that. For instance, it would impact the UDW, and we can probably think of other places.

While it is a good idea to make this configurable, an option would be to define the siteaccess aware settings, internally, and expose them once we have clarified the uses-cases and covered all the aspects of the feature.